bin/crew

```diff
@@ -1294,6 +1311,13 @@ def resolve_dependencies
   abort "Package #{@pkg.name} is not compatible with your device architecture (#{ARCH}) :/".lightred unless @device[:compatible_packages].any? {|elem| elem[:name] == @pkg.name }
 
   @dependencies = []
+  if @pkg.gnome?
+    GNOME_DEPS.each do |dep|
+      return if @pkg.name == dep
```

Comment:
Should this be `continue if`?

Reply:
This is meant to NOT add the package itself to its own dependencies, but I forget if logic for that already exists... @supechicken?

Reply:
Also, I'm not actually sure if this logic block adds these `GNOME_DEPS` from `const.rb` to the dependencies list for packages marked with `gnome`. Which is the goal here.

Reply:
I guess I'm not understanding.  There are 3 `GNOME_DEPS` but this could abort after the second or third loop and miss any further dependencies.

Reply:
Fixed.

Reply:
I think `next` is better than `continue` (in the meaning of the word itself) (although they do the same thing)